scalafix/src/main/scala/org/typelevel/sbt/TypelevelScalafixPlugin.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2022 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.typelevel.sbt
+
+import sbt._, Keys._
+import scalafix.sbt.ScalafixPlugin
+import ScalafixPlugin.autoImport._
+
+object TypelevelScalafixPlugin extends AutoPlugin {
+
+  override def requires = ScalafixPlugin
+
+  override def trigger = allRequirements
+
+  override def buildSettings = Seq[Setting[_]](
+    semanticdbEnabled := true,
+    semanticdbVersion := scalafixSemanticdb.revision,
+    scalafixScalaBinaryVersion := CrossVersion.binaryScalaVersion(scalaVersion.value),
```

Comment:
Question: why doesn't the scalafix plugin do these settings?

Reply:
I'm really not sure - I suppose they are thinking about folks who only want to enable Scalafix selectively, but surely they would just disable the Scalafix plugin instead